packages/web/src/common/store/social/collections/sagas.ts

```diff
@@ -334,10 +334,16 @@ export function* saveCollectionAsync(
   })
   yield* put(event)
 
+  const saveMetadata = action.isFeed
+    ? // If we're on the feed, and the content
+      // being saved is a repost
+      { is_save_of_repost: collection.followee_reposts.length !== 0 }
+    : { is_save_of_repost: false }
```

Comment:
can this just be undefined?

Reply:
yea could be undefined! i figured it's best to avoid passing undefined's when possible, and passing in no metadata to the backend defaults to is_save_of_repost being false anyways, so making the default explicit seemed cleanest. if you think otherwise and prefer undefined tho lmk!